react-exercises/modal-react/src/Components/Modal/index.js

```diff
@@ -63,6 +73,9 @@ Modal.propTypes = {
   cancelText: PropTypes.string,
   okButtonProps: PropTypes.object,
   cancelButtonProps: PropTypes.object,
+  size: PropTypes.string,
```

Comment:
use PropTypes.oneOfType([]),
```suggestion
  size: size: PropTypes.oneOfType([
    PropTypes.number,
    PropTypes.oneOf([ 'medium', 'large', 'etc ]),
]),,
```